docs/articles.bib

```diff
@@ -809,6 +809,13 @@ @misc{WikiCNOT
   
 }
 
+@misc{WikiCyclicPermutation,
+  author  = "Wikipedia",
+  title   = "Cyclic permutation",
+  howpublished = {https://en.wikipedia.org/wiki/Cyclic_permutation}
+  
+}
+
```

Comment:
minor not: Same as what @vprusso said, you might want to move `WikiCyclicPermutation` to after L831